setup.py

```diff
@@ -60,9 +60,12 @@
         "datasets",
         "accelerate>=0.20.3,!=1.1.0",
         "pynvml",
-        "compressed-tensors"
-        if version_info.build_type == "release"
-        else "compressed-tensors-nightly",
+        "pillow",
```

Comment:
Can we make this a dev dependency rather than a base dependency?

Reply:
i've leveraged optional dependencies a lot in the past, it's a mixed bag though where users are unsure which optional deps they might need and there isn't a whole lot of benefit in not installing `pillow` which appears pretty lightweight and minimal overhead, and is commonly used. I didn't see any precedent for optional deps so i left as required for now. I think it's something we could/should leverage in future though if we have clear separation of optional dep groups

Reply:
This is used by the processor right?

Reply:
yes, i think any of the vision language processors will likely need it

Reply:
leaving as required for now, we may want to split all things multimodal into an optional dep later on but for now it's pretty lightweight